views/partials/header.handlebars

```diff
@@ -5,8 +5,7 @@
   <ul class="navlist">
 
     <li class="navitem"><a href="/">Home</a></li>
-    <li class="navitem"><a href="#">Litters</a></li>
-    <li class="navitem"><a href="#">Meet the Parents</a></li>
+    <li class="navitem"><a href="/breeder/">Meet the Parents</a></li>
     <li class="navitem"><a href="blog">Blog</a></li>
     <li class="navitem"><a href="contact">Contact</a></li>
     <li class="navitem"><a href="FAQ">FAQ</a></li>
```

Comment:
All these `href`'s need a `/` in front of them, for example:

```html
<li class="navitem"><a href="FAQ">FAQ</a></li>
```

needs to be changed to

```html
<li class="navitem"><a href="/FAQ">FAQ</a></li>
```

Trying to click on anything in the navbar from the breeder page doesn't work otherwise.